style.css

```diff
@@ -0,0 +1,998 @@
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  text-align: center;
+  color: #ebebff;
+  padding: 0%;
+  margin-top: -20px;
+}
+
+.footer-container h1 {
+  font-family: var(--body-font);
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+  padding: 0;
+}
+
+.skills ul {
+  list-style: none;
+}
+
+.languages {
+  height: 24px;
+  right: 78px;
+  top: calc(50% - 24px / 2);
+  font-family: var(--body-font);
+  font-style: normal;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 24px;
+  display: flex;
+  color: #344563;
+  padding: 0;
+}
+
+.frame {
+  font-family: var(--body-font);
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  color: #000;
+}
+
+.skl {
+  font-family: var(--body-font);
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  color: #000;
+}
+
+.sub-lang img {
+  width: 48px;
+  height: 48px;
+  left: 12px;
+  border: 1px solid #fff;
+  align-items: center;
+  margin-top: 10px;
+  padding: 0;
+}
+
+.language ul li {
+  display: flex;
+  margin: 36px 0;
+  flex-direction: row;
+  padding: 12px;
+  margin-right: 2rem;
+  border-radius: 7px;
+  gap: 12px;
+  align-items: center;
+  top: 72px;
+}
+
+.arrow {
+  width: 20px;
+  height: 20px;
+  margin-top: 5px;
+  margin-left: 150px;
+}
+
+.arrow2 {
+  width: 20px;
+  height: 20px;
+  margin-top: 5px;
+  margin-left: 220px;
+}
+
+.social-media2 {
+  margin-left: -9%;
+}
+
+.line2 {
+  margin-top: 30px;
+  border-top: 1px solid #dfe1e6;
+}
+
+form {
+  margin-top: -30px;
+  display: flex;
+  margin-right: 11%;
+  padding: 20px;
+  align-items: center;
+}
+
+.footer li input {
+  border: 1px solid #cfd8dc;
+  width: 290px;
+  margin-top: 10px;
+  border-radius: 8px;
+  padding: 24px;
+}
+
+.footer textarea {
+  padding: 24px;
+}
+
+.footer li textarea {
+  border: 1px solid #cfd8dc;
+  height: 180px;
+  width: 290px;
+  margin-top: 10px;
+  border-radius: 8px;
+}
+
+.footer li button {
+  border: 1px solid #fff;
+  padding: 12px 16px;
+  border-radius: 8px;
+  font-family: "poppins", sans-serif;
+  font-size: 17px;
+  font-weight: 500;
+  color: #6070ff;
+  cursor: pointer;
+  transition: 1.2s all;
+}
+
+.footer li button:active {
+  transform: scale(0.98);
+  box-shadow: 3px 2px 22px 1px #0000003d;
+  background: #2230d2;
+  border-radius: 8px;
+}
+
+.footer li button:hover {
+  transform: scale(1.05);
+  background: #6070ff;
+  color: white;
+  box-shadow: 0 8px 16px #4053fc3d;
+  border-radius: 8px;
+}
+
+.log {
+  transition: all 1s ease-in-out;
+}
+
+.log:hover {
+  background-color: transparent;
+  transform: rotate(180deg);
+}
+
+.animation {
+  transition: all 1s ease;
+}
+
+.animation:hover {
+  transform: scale(2);
+}
+
+@media screen and (min-width: 768px) {
```

Comment:
- [ ] i can see you did well to meet the desktop version design requirements good job so far 👍 but there is something you are still missing out , kindly consider making sure that your mobile version design stays the same as you build the desk top version this will make your project meet all the design requirements  check on the screen shoots below and make sure you  fix them 😃 
- your mobile vesrions design which were damaged 👇 

1.
<img width="447" alt="Screen Shot 2023-03-31 at 14 49 30" src="https://user-images.githubusercontent.com/60146030/229125068-99c520b2-d18b-45df-9efa-73e0226f5066.png">

2. 
<img width="323" alt="Screen Shot 2023-03-31 at 14 53 37" src="https://user-images.githubusercontent.com/60146030/229125505-8cef9784-029c-42c6-a531-82d8034db5ee.png">

3. 
<img width="380" alt="Screen Shot 2023-03-31 at 14 54 55" src="https://user-images.githubusercontent.com/60146030/229125832-3a386f85-eee8-40ae-a616-5dd852226518.png">

4.
<img width="380" alt="Screen Shot 2023-03-31 at 14 56 03" src="https://user-images.githubusercontent.com/60146030/229126160-a2ba8837-4321-4218-8c10-de2151508dbd.png">
 

5.
<img width="380" alt="Screen Shot 2023-03-31 at 14 57 48" src="https://user-images.githubusercontent.com/60146030/229127136-76daa203-917b-4e9a-9576-e19393010b26.png">

6.   
<img width="380" alt="Screen Shot 2023-03-31 at 15 00 28" src="https://user-images.githubusercontent.com/60146030/229127199-f44a9547-1911-438d-990b-11c71723e1a6.png">

7. 
<img width="281" alt="Screen Shot 2023-03-31 at 15 02 34" src="https://user-images.githubusercontent.com/60146030/229127482-a865f618-871c-4015-b926-f7addbef6710.png">


kindly consider comparing each screenshoot i added above with the figma design and make sure you fix everything i highlighted